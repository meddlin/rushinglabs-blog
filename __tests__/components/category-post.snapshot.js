import { render } from '@testing-library/react';
import CategoryPost from '../../components/category-post';

it('renders homepage unchanged', () => {
    const testObj = {
        id: "this-is-a-test", 
        year: "2022",
        date: "2022-11-25", 
        title: "This is a Test", 
        preview: "", 
        previewImage: "", 
        previewImageWidth: "", 
        previewImageHeight: "", 
        previewImageCreditText: "",
        previewImageCreditUrl: "",
        section: "" 
    }
    const { container } = render(<CategoryPost {...testObj} />);

  expect(container).toMatchSnapshot()
})