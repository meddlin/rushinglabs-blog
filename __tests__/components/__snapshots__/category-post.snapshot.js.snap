// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <li
    class="listItem"
  >
    <div
      class="justify-center items-center"
    >
      <div
        class="justify-center items-center"
      >
        <div
          class="flex flex-col"
        >
          <div />
          
        </div>
      </div>
      <a
        href="/blog/2022/this-is-a-test"
      >
        This is a Test
      </a>
    </div>
    <br />
    <small
      class="subpreview"
    >
      
      <time
        datetime="2022-11-25"
      >
        November 25, 2022
      </time>
    </small>
    <p />
    <a
      class="readMoreLink"
      href="/blog/2022/this-is-a-test"
    >
      Read More â€”&gt;
    </a>
  </li>
</div>
`;
